src/components/Input.jsx

```diff
@@ -1,5 +1,21 @@
-const Input = () => {
-  return <div>Input</div>;
+const Input = ({ value, onChange, name, placeholder, classes }) => {
+  return (
+    <div className={`flex items-center w-1/2 text-white ${classes}`}>
+      <p className="text-xs px-1 z-40 bg-code-black flex fixed ml-3 mb-[3%] select-none">
```

Comment:
why do u need z-40? what's the purpose of fixed and flex, dont think you need fixed. remove the select-none

Reply:
the fixed stops the label from displacing the input field